Include/typeslots.h

```diff
@@ -88,3 +88,51 @@
 /* New in 3.5 */
 #define Py_tp_finalize 80
 #endif
+#define Py_tp_name 81
+#define Py_tp_basicsize 82
+#define Py_tp_itemsize 83
+#define Py_tp_vectorcall_offset 84
+#define Py_tp_flags 85
+#define Py_tp_weaklistoffset 86
+#define Py_tp_dict 87
+#define Py_tp_dictoffset 88
+#define Py_tp_mro 89
+#define Py_tp_cache 90
+#define Py_tp_subclasses 91
+#define Py_tp_weaklist 92
+#define Py_tp_nb_add 93
+#define Py_tp_nb_subtract 94
+#define Py_tp_nb_multiply 95
+#define Py_tp_nb_remainder 96
+#define Py_tp_nb_divmod 97
+#define Py_tp_nb_power 98
+#define Py_tp_nb_negative 99
+#define Py_tp_nb_positive 100
+#define Py_tp_nb_absolute 101
+#define Py_tp_nb_bool 102
+#define Py_tp_nb_invert 103
+#define Py_tp_nb_lshift 104
+#define Py_tp_nb_rshift 105
+#define Py_tp_nb_and 106
+#define Py_tp_nb_xor 107
+#define Py_tp_nb_or 108
+#define Py_tp_nb_int 109
+#define Py_tp_nb_reserved 110
+#define Py_tp_nb_float 111
```

Comment:
What's the difference between `Py_tp_nb_float` and `Py_nb_float` (11)?

Reply:
Oh, no difference. I copy this slots from old commit~.

Reply:
Oh, wait~
`PyHeapTypeObject` have `as_number` and `PyTypeObject` have  `tp_as_number`, so it is need  two vars, no?

Reply:
You can use ` tp_as_number` in both cases.
`PyHeapTypeObject` also have `as_number` directly included, but ` tp_as_number` points to it.

See: https://github.com/python/cpython/blob/345cd37abe324ad4f60f80e2c3133b8849e54e9b/Objects/typeobject.c#L2984-L2989

Reply:
oh, you are right.

Reply:
> You can use tp_as_number in both cases.

Looks like there have a block case in https://github.com/python/cpython/blob/master/Objects/typeobject.c#L3033.

For example, if the `slot->slot`'s value is 1(Py_bf_getbuffer), we should use `{Py_bf_getbuffer, offsetof(PyBufferProcs, bf_getbuffer), offsetof(PyHeapTypeObject, as_buffer)` to mark Py_bf_getbuffer's offset and update it in L3033 of typeobject.c. But we can not get the static type's `bf_getbuffer`  by using this offset(`{Py_bf_getbuffer, offsetof(PyBufferProcs, bf_getbuffer), offsetof(PyHeapTypeObject, as_buffer)`)